docs/reference/restructuredtext/text-roles.rst

```diff
@@ -10,6 +10,19 @@ You can also :doc:`add your own custom text roles </extension/text-roles>`.
 
 Currently the following text roles are implemented:
 
+.. phpdoc:class-list:: [?(@.interfacesIncludingInherited contains "\phpDocumentor\Guides\RestructuredText\TextRoles\TextRole")]
```

Comment:
Text roles do not have to have a class. Most textroles without special needs only have a template

Reply:
hm.. that's unfortunate, let's see how to fix this.

Reply:
Yes, lets fix it in a follow up